plugins/aea-ledger-fetchai/setup.py

```diff
@@ -42,13 +42,7 @@
             "test_tools/data/*",
         ]
     },
-    install_requires=[
-        "open-aea>=1.0.0, <2.0.0",
-        "ecdsa>=0.15,<0.17.0",
-        "bech32==1.2.0",
-        "pycryptodome>=3.10.1,<4.0.0",
-        "cosmpy>=0.2.0",
```

Comment:
I would keep these explicit dependencies in addition to the new one. All direct dependencies should be listed imo to ensure no surprises when changes occur.

Reply:
@DavidMinarsch 
no point to list it multipe times.
cause all the deps have to be in sync,
change in one place will force to change it in another. double work. and possible place for conflicts